lib/config/app_config.dart

```diff
@@ -21,6 +21,10 @@ abstract class AppConfig {
   static double bubbleSizeFactor = 1;
   static double fontSizeFactor = 1;
 
+  static String registrationUrl = 'https://register.tom-dev.xyz/';
+
+  static String twakeWorkplaceHomeserver = 'https://matrix.tom-dev.xyz/';
```

Comment:
why not use const

Reply:
we should change it if config.json have other value